docs/configuration_files.md

```diff
@@ -735,21 +735,25 @@ This task is an upgrade version of the MoveToMapPokemon task. It will fetch poke
 ### Options
 [[back to top](#table-of-contents)]
 
+* `sources` - Defines whether the **WHOLE** task is enabled or not. Please bear in mind that even if the task is enabled, all or any of its sources can be disabled. (default: false)
```

Comment:
"_`sources` - Defines whether the **WHOLE** task is enabled or not. Please bear in mind that even if the task is enabled, all or any of its sources can be disabled. (default: false)_" and "_`sources.enabled` - Defines whether this source is enabled or not. This has nothing to do with the task's `enabled`_"are two things completely different. Does this really sound confusing to you? Really?